sgftools/leela.py

```diff
@@ -224,6 +224,18 @@ def reset(self):
         """
         self.send_command('clear_board')
 
+    def save_sgf(self, sgf_fn):
+        """
+        Save sgf to checkpoint file
+        """
+        self.send_command('printsgf %s' % sgf_fn)
+
+    def load_sgf(self, sgf_fn):
```

Comment:
right now it's usage is not implemented yet?

Reply:
Nope